src/lbaf/Execution/lbsAlgorithmBase.py

```diff
@@ -2,7 +2,7 @@
 import sys
 from logging import Logger
 
-from ..IO.lbsStatistics import compute_function_statistics
+from ..IO.lbsStatistics import compute_function_statistics, Statistics
```

Comment:
Remove unused import:
```suggestion
from ..IO.lbsStatistics import compute_function_statistics
```